### DISCUSSION TITLE
Issue with redirect in livewire 3 docs

### DISCUSSION THREAD

**Discussion by eliezerben** [livewire/livewire#8128] [1. Help] [answered]
Hi,

I'm learning livewire through the docs at livewire.laravel.com.

Right now I'm trying this example: https://livewire.laravel.com/docs/forms#submitting-a-form
But this line doesn't seem to work: 
```php
return $this->redirect('/posts')
            ->with('status', 'Post successfully created.');
```
I get the error
`Call to a member function with() on null`

As far as I can see, the [redirect method](https://github.com/livewire/livewire/blob/c65b3f0798ab2c9338213ede3588c3cdf4e6fcc0/src/Features/SupportRedirects/HandlesRedirects.php#L9) doesn't return anything. 

Is it an error in the docs or am I missing something?

Thanks.

**Comment by danie-ramdhani** [comment, marked as answer]
Another one that mismatched in documentation? 😅 (idk is that `with` function is exists or mismatch with with documentation)

use `session` helper might work.

```php
session()->flash('status', 'Post successfully updated.');
```

**Comment by joshhanley** [comment]
@eliezerben yeah I think @danie-ramdhani is correct that there is an issue with the docs are there is no `->with()` method on Livewire's native redirect method.

But if you use Laravel's it should work `return redirect('/posts')->with('status', 'Post successfully created.');`.

PRs to fix the docs are welcome 🙂

**Reply by danie-ramdhani** [reply]
done https://github.com/livewire/livewire/pull/8131
